API/utils/ConnectionSettings.php

```diff
@@ -9,10 +8,22 @@
 $websocketPort = '8010';
 $externalWebsocketPort = '443';
 $externalWebsocketHost = 'wss.ttapps.org';
+define('ROOT_PATH', dirname(__DIR__) . '/../');
```

Comment:
The only usages of ROOT_PATH and $filesRoot are in tests/bootstrap. Can you move it there instead? Not sure why it's here if it's not connection related nor necessary for non-test environments.